.github/workflows/publish.yaml

```diff
@@ -31,7 +31,7 @@ jobs:
 
       - name: Publish to PyPI
         if: startsWith(github.event.ref, 'refs/tags')
-        uses: pypa/gh-action-pypi-publish@master
+        uses: pypa/gh-action-pypi-publish@v1.8.8
```

Comment:
Similar vein as the other question: why are we using a particular branch that's not the main branch?

Reply:
The main/master branch has been "sunset" and is no longer receiving updates (according to the docs of the action). I can add a dependabot action that will auto-update this tag via PRs in the future.

Reply:
gotcha